src/components/Editor.js

```diff
@@ -150,6 +152,30 @@ const Editor = React.createClass({
     }
   },
 
+  toggleBreakpointDisabledStatus(line) {
```

Comment:
Hmm, I bet we can avoid adding this function
either

a)
```js
toggleBreakpoint(line, {disable} = {}) {
}
```

This way, callee's can say that they'd like to disable a bp instead of removing it. 

or b)

add a disableBreakpoint function that's called in that case and otherwise we call `toggleBreakpoint`

```js
disableBreakpoint(line) {
}
```

just a thought... could be wrong :)